src/styles/app/37-database.css

```
@@ -1402,35 +1491,83 @@
-	color: color-mix(in srgb, var(--database-tone) 52%, var(--text-primary));
-	overflow: hidden;
-	text-overflow: ellipsis;
+	color: var(--pill-contrast-fg);
 	white-space: nowrap;
-	flex: 0 1 auto;
+	flex: 0 0 auto;
 }
 
 .databaseCellPillMore {
 	--database-tone: var(--text-tertiary);
 	flex: 0 0 auto;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)